CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.5.2
+
+* Enhancements
+  * Added `NERVES_DATA_DIR` environment variable for specifying the location
+    for global artifacts and downloads.
+  * Configure `NERVES_DATA_DIR` to `XDG_DATA_HOME` if defined.
+  * Add `nerves` config for `source_date_epoc` to enable reproducible builds.
+  * Allow `erlinit.config` options to be overridden from Mix config.
+    The options from the system are merged with the options specified in the
+    config for `:nerves, :erlinit`.
+
+* Bug fixes
+  * Improved support for identifying Windows Subsystem for Linux in mix task
```

Comment:
This seems like an improvement rather than a bug fix.
  * Windows Subsystem for Linux improvements